pkg/crc/ssh/client.go

```diff
@@ -105,6 +106,10 @@ func (client *NativeClient) Run(command string) ([]byte, []byte, error) {
 	session.Stderr = &stderr
 
 	err = session.Run(command)
+	if err != nil {
+		client.stale = true
+		client.conn.Close()
```

Comment:
With `client.conn = nil` after this, it looks like `client.stale` is not needed?